app/assets/stylesheets/donor.css.scss

```diff
@@ -189,9 +187,15 @@
 }
 
 .donation-item {
-  background: $donor-profile-light-grey;
-  margin: 10px 0;
+  background: $light-grey;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets